lldb/include/lldb/Core/Telemetry.h

```diff
@@ -93,64 +142,84 @@ class TelemetryManager : public llvm::telemetry::Manager {
 public:
   llvm::Error preDispatch(llvm::telemetry::TelemetryInfo *entry) override;
 
-  const llvm::telemetry::Config *GetConfig();
+  /// Returns the next unique ID to assign to a command entry.
+  int MakeNextCommandId();
```

Comment:
Do the commands always need to start at zero? Or is it sufficient for this to be monotonically increasing? This really feels like an implementation detail of `CommandInfo`, so if you just need the latter, I'd use a global static that gets incremented.

Reply:
done